sql/ee/dg/archive_dest_status.sql

```diff
@@ -0,0 +1,7 @@
+col destination format a50;
```

Comment:
Please add a small header:
--
-- Kennart J.
-- Version: 0.1
-- Date: 23.05.2023
--
-- Status from v$archive_dest_status

Reply:
and finally add a line to _scriptinfo.txt in same folder.